source/cloud-service/archive-data/access.rst

```diff
@@ -14,28 +14,32 @@ To access your archive data, you need an AWS token that grants permission on the
       See the :doc:`Wazuh Cloud CLI </cloud-service/cli/index>` section to learn how to list and download your archive data automatically.
 
 
-The following example describes the steps to follow to list the files of your archive data:
+Generate your API key
+---------------------
 
+To obtain the api key token it is necessary to execute this step described in the `Wazuh Cloud CLI </cloud-service/cli/index>` section.
 
-1. Before your start using the Wazuh Cloud API, you need an API key. To generate your API key, see the :ref:`Authentication <cloud_apis_auth>` section.
 
-2. Use the ``POST /storage/token`` endpoint of the :cloud-api-ref:`Wazuh Cloud API <tag/storage>` to get the AWS token and access the archive data of a specific environment. In this example, we generate an AWS token valid for 3600 seconds for environment `0123456789ab`.
+Get the AWS token
+------------------
+
+Use the ``POST /storage/token`` endpoint of the :cloud-api-ref:`Wazuh Cloud API <tag/storage>` to get the AWS token and access the archive data of a specific environment. In this example, we generate an AWS token valid for 3600 seconds for environment `012345678ab`.
 
    .. code-block::
 
       curl -XPOST https://api.cloud.wazuh.com/v2/storage/token -H "x-api-key: <YOUR_API_KEY>" -H "Content-Type: application/json" --data '
       {
-         "environment_cloud_id": "0123456789ab",
+         "environment_cloud_id": "012345678ab",
          "token_expiration": "3600"
       }'
 
    .. code-block:: console
       :class: output
 
```

Comment:
Please highlight the relevant information lines with `:emphasize-lines:`, for example

```
   .. code-block:: console
      :class: output
      :emphasize-lines: 7-9

      {
         "environment_cloud_id": "012345678ab",
         "aws": {
            "s3_path": "wazuh-cloud-cold-us-east-1/012345678ab",
            "region": "us-east-1",
            "credentials": {
               "access_key_id": "mUdT2dBjlHd...Gh7Ni1yZKR5If",
               "secret_access_key": "qEzCk63a224...5aB+e4fC1BR0G",
               "session_token": "MRg3t7HIuoA...4o4BXSAcPfUD8",
               "expires_in": 3600
            }
         }
      }
```